<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>250</number>
     </property>
     <property name="topMargin">
      <number>300</number>
     </property>
     <property name="rightMargin">
      <number>250</number>
     </property>
     <property name="bottomMargin">
      <number>300</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);

border-radius : 10px;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="2,5,5">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">color : rgb(0, 0, 0);
background-color : rgba(255, 255, 255, 0);

font-family : 'Colonna MT';
font-weight : bold;
font-size : 40pt; </string>
               </property>
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Date&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="member_num">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0, 0, 0, 50);
color : rgb(255, 255, 255);

font-family : 'Dosis';
font-size : 12pt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
